grice/db_controller.py

```diff
@@ -212,15 +222,16 @@ def table_api(self, name):
     table_api.methods = ['GET', 'POST']
 
     def query_api(self, name):
-        column_names, page, per_page, filters, sorts, join = parse_query_args(request.args)
+        column_names, page, per_page, filters, sorts, join, group_by = parse_query_args(request.args)
 
         try:
             table_info = self.db_service.get_table(name)
         except NotFoundError as e:
             return jsonify(success=False, error=str(e)), 404
 
         try:
-            rows, columns = self.db_service.query_table(name, column_names, page, per_page, filters, sorts, join)
+            rows, columns = self.db_service.query_table(name, column_names, page, per_page, filters, sorts, join, group_by, 
+                format_as_list=request.args.get('_list', '').lower() in ['t', 'true', '1'])
```

Comment:
NB: got a change coming where all these parameters are in one object, so we can handle POSTed JSON.